manifests/base/paddock/paddock-settings.yaml

```diff
@@ -1,19 +1,18 @@
 kind: Secret
 apiVersion: v1
 metadata:
-    name: paddock-settings
+  name: paddock-settings
 stringData:
-    SECRET_KEY: 7a54cf2d6fce83d7de860cf2a82e7f49b9d3045f
-    DJANGO_SUPERUSER_PASSWORD: 1a9fce4d655c94d66b9ea2aec85380d8ad54a790
-    DJANGO_SUPERUSER_EMAIL: info@b4mad.racing
-    DJANGO_SUPERUSER_USERNAME: admin
-    B4MAD_RACING_MQTT_USER: crewchief
-    B4MAD_RACING_MQTT_PASSWORD: crewchief
-    B4MAD_RACING_MQTT_HOST: mosquitto-tcp
-    B4MAD_RACING_MQTT_PORT: "1883"
-    B4MAD_RACING_INFLUX_ORG: b4mad
-    B4MAD_RACING_INFLUX_TOKEN: 41e593a879202f9ebc0e5bc6f1b18a40e0a0d855
-    B4MAD_RACING_INFLUX_URL: http://influxdb2/
-    READONLY_DATABASE_URL: postgresql://reader:995f43ad3dc7ce93c4ae7504b4d7c229deb7c5ae@db-primary:5432/paddock
```

Comment:
This file is not even applied, how got the ENV created?

Reply:
it's created in `env/phobos` - this is just here for reference